plugins/aws/test/resources/service_quotas_test.py

```diff
@@ -40,10 +40,10 @@ def test_vpc_quotas() -> None:
     expect_quotas(builder, 3)
 
 
-def test_alb_quotas() -> None:
-    _, builder = round_trip_for(AwsServiceQuota, "usage", "quota_type")
-    AwsAlb.collect_resources(builder)
-    expect_quotas(builder, 2)
+# def test_alb_quotas() -> None:
```

Comment:
Moved to draft because got `RuntimeError: cannot schedule new futures after shutdown`